<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>solutionUI</class>
 <widget class="QDialog" name="solutionUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Результаты</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Pictures/toinst.jpg</normaloff>../../Pictures/toinst.jpg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Режим камеры</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QRadioButton" name="freeCamera">
              <property name="text">
               <string>Свободная камера</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="oxy">
              <property name="text">
               <string>Плоскость Oxy</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="oxz">
              <property name="text">
               <string>Плоскость Oxz</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="ozy">
              <property name="text">
               <string>Плоскость Ozy</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Визуализация</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="BoxScene" name="openGLWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>400</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Размещение ящиков</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Лог</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="logTextEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BoxScene</class>
   <extends>QOpenGLWidget</extends>
   <header>boxscene.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
